city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,143 @@
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookJusticeAdvisorySpider(CityScrapersSpider):
+    name = "cook_justice_advisory"
+    agency = "Cook County Justice Advisory"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cookcountyil.gov"]
+
+    def start_requests(self):
```

Comment:
This is the default behavior for `start_requests` when `start_urls` is supplied, so the URL can be moved into `start_urls` and this method removed